lib/modules/datasource/rubygems/__fixtures__/contribsys-com-versions.txt

```diff
@@ -0,0 +1,4 @@
+created_at: 2022-06-15T17:10:25+00:00
```

Comment:
please inline this fixture and reduce versions, we don't need so many for testing

Reply:
Is there somewhere that new conventions are being tracked? Like a best practices doc? Because there are a few patterns that I found like this that were just replicating things that were already in the codebase. If those patterns are not correct, it might be good to document those choices for uninitiated devs like myself.